### DISCUSSION TITLE
为什么我设备有告警，设备状态还是显示正常呢？

### DISCUSSION THREAD

**Discussion by TT0404** [apache/hertzbeat#1839] [General]
![image](https://github.com/apache/hertzbeat/assets/91655580/a3fc9f37-06f0-4574-8063-61a2d7374eb2)
![image](https://github.com/apache/hertzbeat/assets/91655580/a6b05329-157d-41a0-a53a-7ee481c54aa2)

**Comment by tomsun28** [comment]
Hi, because warning alarms will not affect the monitoring status, only the corresponding availability threshold alarm will affect it. Btw, It is recommended to communicate in English.
